src/test/java/no/nav/fo/veilarbregistrering/bruker/FoedselsnummerTestdataBuilder.java

```diff
@@ -0,0 +1,11 @@
+package no.nav.fo.veilarbregistrering.bruker;
+
+public class FoedselsnummerTestdataBuilder {
```

Comment:
Planen er å samle flere Foedselsnummer her - vet vi har noen fiktive som brukes, og også støtte for generering "on-the-fly". Men at det er litt "vanskelig" å finne fra gang til gang.